src/table/Exposure.cc

```diff
@@ -186,61 +186,72 @@ void ExposureFitsWriter::_writeRecord(BaseRecord const & r) {
 //----- ExposureFitsReader ---------------------------------------------------------------------------------
 //-----------------------------------------------------------------------------------------------------------
 
-// A custom FitsReader for ExposureTable/Record - this gets registered with name EXPOSURE, so it should get
-// used whenever we read a table with AFW_TYPE set to that value.
-
-namespace {
-
-class ExposureFitsReader : public io::FitsReader {
+// FitsColumnReader that reads a Persistable subclass T (Wcs, Psf, or Calib here) by using an int
+// column to retrieve the object from an InputArchive and attach it to an ExposureRecord via
+// the Setter member function pointer.
+template <typename T, void (ExposureRecord::*Setter)(PTR(T const))>
+class PersistableColumnReader : public io::FitsColumnReader {
```

Comment:
Naming things is hard, but:

At first glance, this sounds like a ``FitsColumnReader`` which can be persisted. In fact, I think it's a ``FitsColumnReader`` which is used to read things which are persistable. Not a big deal (and maybe this is actually following naming conventions elsewhere), but I wonder if the name could be more explicit?